scripts/index.js

```diff
@@ -1,2 +1,162 @@
 // add your solutions here
-// testing, testing, 123
+// Function Exercises
+
+// Positive Numbers
+
+function positiveArray(arr) {
+  //   newArray = [];
+  //   arr.forEach(function(num) {
+  //     if (num > 0) {
+  //       newArray.push(num);
+  //     //   console.log(num);
+  //     }
+  //   });
+  //   return newArray;
+
+  return arr.filter(arr => arr > 0);
+  //   console.log(positiveArray([1, 2, 3, -5, 6, -4]));
+}
+
+// Even Numbers
+function evenNumbers(arr) {
+  return arr.filter(arr => arr % 2 == 0);
+  //   console.log(evenNumbers([1, 2, 3, 4, 5, 7, 9, 10, 12, 16]));
+}
+
+// Square the Numbers
+function squaredNumbers(arr) {
+  squaredArray = arr.map(num => num * num);
+  return squaredArray;
+}
+// console.log(squaredNumbers([1, 2, 3, 4, 5]));
+
+// Cities 1
+
+const cities = [
+  { name: "Los Angeles", temperature: 60.0 },
+  { name: "Atlanta", temperature: 52.0 },
+  { name: "Detroit", temperature: 48.0 },
+  { name: "New York", temperature: 80.0 }
+];
+
+// function coolerThan70(arr) {
+//   coolCities = [];
+//   arr.forEach(city => {
+//     if (city.temperature < 70) {
+//       coolCities.push(city);
+//     }
+//   });
+
+//   return coolCities;
+// }
+
+function coolerThan70(arr) {
+  return arr.filter(city => city.temperature < 70);
+}
+coolerThan70(cities);
+
+// Cities 2
+
+function returnCityNames(arr) {
+  nameArray = arr.map(city => city.name);
```

Comment:
Make sure to use `const` or `let` when you assign to a variable for the first time. Check out http://tobyho.com/2011/10/25/js-accidental-global-variables/ for more info.